src/generic/datavgen.cpp

```diff
@@ -910,6 +910,33 @@ class wxDataViewMainWindow: public wxWindow
     // assumes that all columns were modified, otherwise just this one.
     bool DoItemChanged(const wxDataViewItem& item, int view_column);
 
+    // Return whether the item has more than one column that have values
+    bool IsItemMultivalued(const wxDataViewItem& item) const
```

Comment:
Looking at the code, we always use `!IsItemMultivalued()`, so it seems better to rename this to `IsItemSingleValued()` and remove the negation.

Reply:
I considered this alternative. I opted for the multivalued naming as being marginally more precise because it is theoretically possible for an item to have no values and an item with 0 values should be treated as one with a single value, rather than as one with more than one value. Let me know if you still want me to change this.